website/pages/home/style.module.css

```diff
@@ -34,12 +34,6 @@
 
   & :global(.g-code-block) {
     width: 100%;
-    margin-left: 50px;
-    overflow: scroll;
-
-    @media (max-width: 1000px) {
-      margin-left: 0;
-    }
```

Comment:
I've tweaked the styles in `temporary_text-split` rather than overriding `.g-code-block` - this is also kind of in preparation for conversion of components to CSS modules.